### ISSUE TITLE
Dialog labels not read using chrome + talkback in Incredible Accessible Modal Window

### ISSUE THREAD

**Issue by kwiniarski97** [a11yproject/a11yproject.com#827]
## Summary
When you open _The Incredible Accessible Modal Window, Version 4_ in the Android with talkback it doesn't read dialog label. It reads that you focused first name field and nothing more.
> 1.Turn talkback in Android phone
> 2. Go to https://codepen.io/BeyondHyper/pen/NREjpP
> 3. Open dialog example
> 4. Dialog label is not read

### Expected result
Dialog label should be read announcing user that a dialog has been opened.

### Actual result
Dialog label wasn't read. Talkback read that you are editing field with no indication that dialog has been opened whatsoever. 

### Additional Information
Google Pixel 1. Android 9. Chrome 75.0.3770.143

**Comment by scottaohara** [comment]
Hi @kwiniarski97 

This is an issue with TalkBack and/or Android Chrome, not with the dialog example.

Might you please file this bug with them?  Thanks.
